dm/master/server.go

```diff
@@ -1126,6 +1126,13 @@ func (s *Server) OperateSource(ctx context.Context, req *pb.OperateSourceRequest
 	}
 
 	resp.Result = true
+	// tell user he should use `start-relay` to manually specify relay workers
+	for _, cfg := range cfgs {
+		if cfg.EnableRelay {
+			resp.Msg = "Please use `start-relay` to specify which workers should pull relay log of relay-enabled sources."
```

Comment:
It's confused for the user if they don't need to start-relay in multiple workers.

Reply:
for current behaviour, they must using `start-relay` to pull relay log. This's the only change to tell uesr we changed the behaviour